### PR TITLE
refactor(banners): upgrade component - INNO-594

### PR DESCRIPTION
**Pull request by emeryro**
Refactor of banners.
I had to add background banner (missing), and update blockquotes component as it didn't allow comosition and didn't display quote author.

### CODE REVIEW

**Comment by emeryro** [comment]
Tests added

File: framework/components/ecl-banners/ecl-banners.config.js [hunk 1]
```diff
@@ -1,6 +1,46 @@
...
+    `;
+  },
+  default: 'background',
+  variants: [
+    {
+      name: 'background',
+      label: 'Background banner',
+      context: {
+        modifier: 'background',
+        src: 'http://lorempixel.com/1600/370/abstract/',
```
Comment (degliwe) [inline framework/components/ecl-banners/ecl-banners.config.js]: This will not pass visual regression tests.
Please target always the same image.

**Comment by emeryro** [comment]
Screenshots have been updated, with a fixed background

**Comment by emeryro** [comment]
Conflict solved

**Pull request merged**